plugins/dotplot-view/src/DotplotRenderer/configSchema.ts

```diff
@@ -16,10 +16,20 @@ export default ConfigurationSchema(
       defaultValue: 'black',
       contextVariable: ['feature'],
     },
+    posColor: {
+      type: 'color',
+      description: 'the color for forward alignments',
+      defaultValue: 'purple',
+    },
+    negColor: {
+      type: 'color',
+      description: 'the color for reverse alignments',
+      defaultValue: 'blue',
+    },
```

Comment:
do these descriptions need to be updated? they talk about alignments

Reply:
dotplot plots alignments :)